pkg/docker/index.html

```diff
@@ -143,91 +143,93 @@ <h3>
-            </td>
-          </tr>
-          <tr>
-            <td translate>Command:</td>
-            <td colspan="3" id="container-details-command"></td>
-          </tr>
-          <tr>
-            <td translate>State:</td>
-            <td colspan="3" id="container-details-state"></td>
-          </tr>
-          <tr>
-            <td translate>Restart Policy:</td>
-            <td colspan="3" id="container-details-restart-policy"></td>
-          </tr>
-          <tr>
-            <td translatable="yes">IP Address:</td>
-            <td colspan="3" id="container-details-ipaddr"></td>
-          </tr>
-          <tr>
-            <td translatable="yes">IP Prefix Length:</td>
-            <td colspan="3" id="container-details-ipprefixlen"></td>
-          </tr>
-          <tr>
-            <td translatable="yes">Gateway:</td>
-            <td colspan="3" id="container-details-gateway"></td>
-          </tr>
-          <tr>
-            <td translatable="yes">MAC Address:</td>
-            <td colspan="3" id="container-details-macaddr"></td>
-          </tr>
-          <tr id="container-details-ports-row" hidden>
-            <td translate>Ports:</td>
-            <td colspan="3" id="container-details-ports"></td>
-          </tr>
-          <tr id="container-details-links-row" hidden>
-            <td translate>Links:</td>
-            <td colspan="3" id="container-details-links"></td>
-          </tr>
-          <tr id="container-details-volumes-row" hidden>
-            <td translate>Volumes:</td>
-            <td colspan="3" id="container-details-volumes"></td>
-          </tr>
-          <tr>
-            <td>&nbsp;</td>
-            <td>
-              <!-- hidden bar-row to help set a minimum limit for memory -->
-              <div class="bar-row hidden" graph="container-details" value="0/100000000"></div>
-            </td>
-          </tr>
-          <tr id="container-details-memory-row" class="interactive" hidden>
-            <td translate>Memory usage:</td>
-            <td colspan="2" id="container-details-memory">
-              <div class="bar-row" graph="container-details"></div>
-            </td>
-            <td class="shrink resource-value" id="container-details-memory-text"></td>
-          </tr>
-          <tr id="container-details-cpu-row" class="interactive" hidden>
-            <td translate>CPU usage:</td>
-            <td colspan="2">
-              <span class="cpu-usage"></span>
-            </td>
-            <td class="shrink resource-value">
-              <span class="cpu-shares"></span>
-            </td>
-          </tr>
-          <tr id="container-details-resource-row" hidden>
-            <td></td>
-            <td>
-              <button class="btn btn-default resource-button" data-toggle="modal"
-                      data-target="#container-resources-dialog" translate>Change resource limits</button>
-            </td>
-          </tr>
+          <tbody>
```

Comment:
I don't think we need to fix them in HTML, but consitency can't hurt.

Reply:
Right. I just grepped for all `<table` in pkg/docker and fixed them all, seemed easier than trying to divine from the warning which particular table these applied to.